doc/api/http2.md

```diff
@@ -1701,16 +1727,755 @@ req.end('Jane');
+the headers get flushed.
+
+Example:
+
+```js
+response.statusCode = 404;
+```
+
+After response header was sent to the client, this property indicates the
+status code which was sent out.
+
+### response.statusMessage
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+Status message is not supported by HTTP/2 (RFC7540 8.1.2.4). It returns
+an empty string.
+
+#### response.stream
+<!-- YAML
+added: REPLACEME
+-->
+
+* {http2.Http2Stream}
+
+The [`Http2Stream`][] object backing the response.
+
+### response.write(chunk[, encoding][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `chunk` {string|Buffer}
+* `encoding` {string}
+* `callback` {Function}
+* Returns: {boolean}
+
+If this method is called and [`response.writeHead()`][] has not been called,
+it will switch to implicit header mode and flush the implicit headers.
+
+This sends a chunk of the response body. This method may
+be called multiple times to provide successive parts of the body.
+
+Note that in the `http` module, the response body is omitted when the
+request is a HEAD request. Similarly, the `204` and `304` responses
+_must not_ include a message body.
+
+`chunk` can be a string or a buffer. If `chunk` is a string,
+the second parameter specifies how to encode it into a byte stream.
+By default the `encoding` is `'utf8'`. `callback` will be called when this chunk
+of data is flushed.
+
+*Note*: This is the raw HTTP body and has nothing to do with
+higher-level multi-part body encodings that may be used.
+
+The first time [`response.write()`][] is called, it will send the buffered
+header information and the first chunk of the body to the client. The second
+time [`response.write()`][] is called, Node.js assumes data will be streamed,
+and sends the new data separately. That is, the response is buffered up to the
+first chunk of the body.
+
+Returns `true` if the entire data was flushed successfully to the kernel
+buffer. Returns `false` if all or part of the data was queued in user memory.
+`'drain'` will be emitted when the buffer is free again.
+
+### response.writeContinue()
+<!-- YAML
+added: REPLACEME
+-->
+
+Does nothing. Added for parity with [HTTP/1]().
+
+### response.writeHead(statusCode[, statusMessage][, headers])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `statusCode` {number}
+* `statusMessage` {string}
+* `headers` {Object}
+
+Sends a response header to the request. The status code is a 3-digit HTTP
+status code, like `404`. The last argument, `headers`, are the response headers.
+For compatibility with [HTTP/1](), one can give a human-readable `statusMessage` as the second argument, which will be silenty ignored and emit a warning.
+
+Example:
+
+```js
+const body = 'hello world';
+response.writeHead(200, {
+  'Content-Length': Buffer.byteLength(body),
+  'Content-Type': 'text/plain' });
+```
+
+This method must only be called once on a message and it must
+be called before [`response.end()`][] is called.
```

Comment:
That doesn’t seem right, I read the `write()` docs as saying that `writeHead()` doesn’t need to be called